rpc-jobs/jjb-setup.yml

```diff
@@ -0,0 +1,64 @@
+- job:
+    name: 'Jenkins-Job-Builder'
+    description: Creates and updates jobs with Jenkins Job Builder.
+    logrotate:
+        daysToKeep: 20
+    node: general
+    parameters:
+        - string:
+            name: JOBS
+            description: "Which jobs to update and with what options."
+            default: -r rpc-jobs
+        - bool:
+            name: IGNORE_CACHE
+            description: "Ignore cache when updating jobs."
+        - string:
+            name: JJB_REPO
+            default: https://github.com/rcbops/rpc-gating
+        - string:
+            name: JJB_BRANCH
+            default: master
+    scm:
+        - git:
+            url: $JJB_REPO
+            branches:
+                - $JJB_BRANCH
+            skip-tag: true
+            wipe-workspace: false
+    wrappers:
+      - credentials-binding:
+          - username-password-separated:
+             credential-id: service_account_jenkins_api_creds
+             username: JENKINS_USER
+             password: JENKINS_API_PASSWORD
+    builders:
+        - shell: |
+            scl enable python27 bash
+
+            if [[ ! -d ".venv" ]]; then
+                virtualenv -p /opt/rh/python27/root/usr/bin/python .venv
+            fi
+            source .venv/bin/activate
+
+            pip install jenkins-job-builder
+
+            cat > jenkins_jobs.ini << EOF
```

Comment:
If we put the credential file in the workspace, though, it'll be viewable in plain text from the UI since you can look at the current state of a job's workspace.